CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 CHANGELOG
 =========
+## 0.34.0 / 2020-02-04
+
+* [Deprecated] Alias `dog` script names as `dogshell`. See [#305](https://github.com/DataDog/datadogpy/pull/305). Thanks [dwminer](https://github.com/dwminer).
```

Comment:
Maybe we should say something like: Please start using `dogshell` instead of `dog` command.

Reply:
```suggestion
* [Deprecated] Alias `dog` script names as `dogshell`. Please start using `dogshell` instead of `dog` command. See [#305](https://github.com/DataDog/datadogpy/pull/305). Thanks [dwminer](https://github.com/dwminer).
```